packages/testing-library/testing-environment/src/lynx/ElementPAPI.ts

```diff
@@ -284,7 +284,8 @@ export const initElementTree = () => {
       styles: string | Record<string, string>,
     ) {
       if (typeof styles === 'string') {
-        e.style.cssText = styles;
+        // Same as from https://github.com/jsdom/jsdom/blob/6197af431c46b95622eb4ce9d3e4df3010c66984/lib/jsdom/living/nodes/ElementCSSInlineStyle-impl.js#L8
```

Comment:
Do we need a changeset for this?

Reply:
This change does not affect the user, but a changeset is still good, let me add it to the changeset.